src/test/java/io/reactivex/schedulers/SchedulerTest.java

```diff
@@ -13,21 +13,20 @@
 
 package io.reactivex.schedulers;
 
-import static org.junit.Assert.*;
-
-import java.util.List;
-import java.util.concurrent.TimeUnit;
-
-import io.reactivex.annotations.NonNull;
-import org.junit.Test;
-
 import io.reactivex.*;
 import io.reactivex.Scheduler.Worker;
+import io.reactivex.annotations.NonNull;
```

Comment:
Please completely restore this file before this PR so it doesn't show up as diff at all.

Reply:
I hope I've finally had the order. :)